programs/C/length_string.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+
+//Created a function to calculate length of string.
+int LengthOfString(char StringInput[50]){
```

Comment:
There should be a space before braces

Reply:
Also i think divyam asked to use camel case, which means first word should start with small letter and rest all words should start with a capital letter while naming variables or functions